### ISSUE TITLE
Words breaking into letters

### ISSUE THREAD

**Issue by urban2015** [Jack000/Deepnest#111]
HI,

I want to use this with words but when i attached the letters together the program breaks the words into letters. I have tried using illustrator and inkscape both failed.

Could you help?

**Comment by amunizp** [comment]
Using inkscape maybe this might help: make each word to paths then merge them using the union tool.

**Comment by urban2015** [comment]
I have tried union the letters together and still the letters behave as separate ones. If i import the same svg file into another program the letters are not separate so the program is separating them.

**Comment by amunizp** [comment]
Can you overlap the letters just a tiny bit? After the nesting is done you can then return it to the original letter space.

**Comment by urban2015** [comment]
Yeah but its not so easy to get the letters back after... I tried connecting letters with shapes to make the letters all joined up but after the nest it was impossible to remove the shapes.

**Comment by amunizp** [comment]
Do you mean wrapping the letters with a line? Using outset? Because that was my next suggestion.
